models/pull_review.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+package models
+
+import (
+	"code.gitea.io/gitea/models/db"
+	"code.gitea.io/gitea/modules/timeutil"
+)
+
+type PRReview struct {
```

Comment:
Please add some comment for the struct and it's better to put it into a sub folder of `models` if possible.

Reply:
Okay, I can move it to a subpackage, even if there exists no such for thing for PRs until now.
Should I create the package `models/pulls` or something else?

Reply:
There was some information about packages, https://docs.gitea.io/en-us/guidelines-backend/#future-tasks

However it's not clear enough.

For example:
* Should the `PRReview` use `models/pullrequest` package, or should it use a separate package `models/prreview`?
* Do we preferer abbr in names (`PRReview` vs `PullRequestReview`)?
* Singular or plural for package names?
* Should all new PR follows the guide? strict or loose?

Reply:
Looks like currently there are some dependent PRs .We need to move issues to sub package at first. Otherwise, the new sub package will depend on `models`, it's different from other sub packages.

Reply:
@lunny I have moved the file now into the new `models/pulls` package.
I deviated from the standard here, because `pull` is a too common variable name which would lead to too many conflicts.
In this case, there is no dependency on `models`, only on `models/db` and on `modules/timeutil`.

@wxiaoguang 
> Singular or plural for package names?

That one is mentioned in the docs, see https://docs.gitea.io/en-us/guidelines-backend/#package-name

But what about the other points? Should we use the abbreviated or the complete name, and how should the package be called?
And would you say that some of your points are out of scope for this PR?

Reply:
Yup, some questions are a little off-topic for this PR. However I believe they are general questions for many PRs (including this one), so I just listed them together.

If the document/guideline could be updated accordingly during these discussions, many developers/PRs would benefit from it. One challenge of Gitea is that the document/guideline are still insufficient. For example, is `pulls` acceptable and when it is acceptable, if there are other naming conflicts like `foo` in future, should we use `foos`? I believe if these consensus could be written down, it would make the development easier and avoid arguments.